src/hooks/useProjects.ts

```
@@ -406,382 +412,273 @@ export const useProjects = () => {
+      }
+      
+      const { data: tasksData, error: tasksError } = await supabase
+        .from('tasks')
+        .select('*')
+        .eq('project_id', projectId)
+        .order('created_at', { ascending: true });
+
+      if (tasksError) throw tasksError;
+      
+      const tasksWithSubtasks = await Promise.all(
+        (tasksData || []).map(async (task: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)